moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -160,22 +159,20 @@ void Servo::setCollisionChecking(const bool check_collision)
 
 bool Servo::validateParams(const servo::Params& servo_params) const
 {
-  bool params_valid = true;
-
   auto robot_state = planning_scene_monitor_->getStateMonitor()->getCurrentState();
   auto joint_model_group = robot_state->getJointModelGroup(servo_params.move_group_name);
   if (joint_model_group == nullptr)
   {
     RCLCPP_ERROR_STREAM(LOGGER, "Invalid move group name: `" << servo_params.move_group_name << '`');
-    params_valid = false;
+    return false;
```

Comment:
Just to offer an explanation for why it was like this, Ibrahim was told that a function should only return in one place. I kind of like this new style a bit better myself. Context-dependent, I guess.

Reply:
I just liked that better, but if that was a discussed before I'll revert that change :+1: